src/Perl6/Actions.nqp

```diff
@@ -1427,11 +1427,31 @@ class Perl6::Actions is HLL::Actions does STDActions {
         my $doc := %*COMPILING<%?OPTIONS><doc>;
         if $doc {
             my $block := $*W.push_lexpad($/);
-
-            my $renderer := "Pod::To::$doc";
-
-            my $module := $*W.load_module($/, $renderer, {}, $block);
-
+            my $rx := /
+                       ^
+                       $<name> = ( [ \w | '::' ] + )
+                       [ ':' $<part> = ( [ 'ver' | 'auth' | 'api' ] )
+                        \< ~ \> $<val> = ( .*? ) ]*
+                       $
+                      /;
+            my $s := match($doc, $rx);
+            my %opts := nqp::hash();
+            my @parts := nqp::atkey($s,'part');
+            my @vals := nqp::atkey($s,'val');
+            my $e := nqp::elems(@parts);
+
+            my $i := 0;
+            while $i < $e { nqp::bindkey( %opts, ~ @parts[$i], ~ @vals[$i++] ) };
+
+            my $renderer := ~ nqp::atkey($s,'name');
+            my $module;
+            try {
+                $module := $*W.load_module($/, $renderer, %opts, $block);
+                CATCH {
+                    $renderer := 'Pod::To::' ~ nqp::atkey($s,'name');
+                    $module := $*W.load_module($/, $renderer, %opts, $block);
```

Comment:
What if this fails? Also, you're introducing the ambiguity that @ugexe has pointed out, which means that now Any::Renderer::I::Want could be used here. Do we want this?

Reply:
JJ I think you misunderstood @ugexe, or perhaps I do. 
He was saying, like I was saying, the argument to --doc= should be a fully qualified module.
But the API requires (as Damian Conway stated in his original POD specification) that it should be --doc=HTML.

@AlexDaniel pointed out that =HTML already exists and is documented.

So my suggestion is to test for the fully qualified name, but then test for a Pod::To::xxx name. 

If that also fails, then the compiler fails, as it does now. There is no default specified. It could default to Pod::To::Text but that is NOT yet specified anywhere.